contributing.md

```diff
@@ -224,8 +224,9 @@ you tests to keep things consistent across the entire codebase.
 #### test/functional
 
 Tests that call the appcenter-cli from PowerShell to execute against an App Center internal test environment live here.
```

Comment:
So I guess only people have access to INT will be able to run the test?

Reply:
Wow. You are totally right. I cannot believe I didn't think of that. Thank you! That makes them useless for external people who want to test their work. I'll revisit this PR and add an option to the code to run it against production.